iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -1153,6 +1170,35 @@ class SchedulingBarrier(CustomOp):
     operations: list[Operation]
 
 
+@define_op("atomic_min")
+@dataclass
+class AtomicOp(BinaryOpBase, ABC):
+    """
+    Represents an atomic operation in the graph.
+    Takes in value (register) and buffer (shared/global) as inputs,
```

Comment:
minor: can you update this to say just Register and Memory as those are the input types.